src/actions/actionsConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { workspace } from 'vscode';
+
+const key = "actions";
+
+export const get = (): string[] => {

```

Comment:
1. why do we need a const to expose a function, can't we just export a function directly?

Reply:
discuss later